client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -119,8 +120,10 @@ const Checkout = createReactClass( {
 			this.setState( { previousCart: nextCart } );
 		}
 
+		// Although this is a part of the gsuiteUpsellTake2 A/B test,
+		// the `abtest()` is not called here to make the data more accurate.
 		if (
-			abtest( 'gsuiteUpsell' ) === 'show' &&
+			isNewSite &&
```

Comment:
Javascript strikes back - this check will _always_ be true (and the one on line 310 too), because it's the... selector from `state/sites/selectors`, not the value from props :)
IIRC, we used to have a linter warning for cases like this? Anyway, probably a good idea to rename these (the selector and the value derived from it), so that they don't clash like this.

Reply:
OMG, how have I missed it? :)